payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/MandateTextUI.kt

```diff
@@ -30,7 +30,7 @@ fun MandateTextUI(
         ),
         color = MaterialTheme.stripeColors.placeholderText,
         modifier = Modifier
-            .padding(vertical = 8.dp)
+            .padding(top = 2.dp)
```

Comment:
I don't see any full screen screenshots for this change, how does it look in the context of everything? For both vertical and tabs?

Reply:
It fits the mandates close to the elements. Vertical mode example:
<img src="https://github.com/user-attachments/assets/68cb9a8a-ff33-43ff-b34c-763910f07bd1" width="50%" height="50%" />

I mainly want to remove most of assumption that the `MandateTextElement` makes especially since we will now be stacking two Mandate elements now with card mandates.

Reply:
iOS doesn't make the assumption on Mandate vertical padding either and theirs is also closer

Reply:
Actually looks like most of our elements make this assumption :/